aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md

```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
 
 Now you need to get your service bus connection string. In the Azure portal, select the service bus namespace that you created and click the Access Key icon.
 
-![An image of a plus sign labeled 'Create', an image of a key labeled 'Access Key', and an image of a trash can labeled 'Delete'.](scaleout-with-windows-azure-service-bus/_static/image7.png)
+![Graphic of a plus sign labeled Create and a key labeled Access Key and a trash can labeled Delete.](scaleout-with-windows-azure-service-bus/_static/image7.png)
 
 Copy the connection string to the clipboard, then paste it into the *connectionString* variable.
 
-![Window titled 'Connect to your namespace' with options underneath.](scaleout-with-windows-azure-service-bus/_static/image8.png)
+![Screenshot of Window titled Connect to your namespace with options underneath.](scaleout-with-windows-azure-service-bus/_static/image8.png)
```

Comment:
Let's change the beginning of your sentence to, "Screenshot of a window..."  Or, "Screenshot that shows the Access Key Connect to your namespace dialog box."